test/DownloadDatabase.jl

```diff
@@ -7,7 +9,7 @@ folder_name = "test/datasets"
 const datasetPath = joinpath(dirname(@__DIR__), folder_name)
 
 
-function download_data(url, datasetPath)
+function download_data(url, datasetPath, battery, dataset)
```

Comment:
@Omadzze Thep param name battery seems a bit weird to me. But i also do not get it.
Please update the docstring aswell